source/access/access_using_mobaxterm.rst

```diff
@@ -0,0 +1,177 @@
+.. _access_using_mobaxterm:
+
+Download and setup MobaXterm
+============================
+
+.. note::
+
+    To access the VSC clusters with MobaXterm your private key must be in OpenSSH format.
+    SSH key pairs generated with PuTTY are in another format (.ppk) and thus cannot be used
+    with MobaXterm. If you have created your key pair with PuTTY, you can
+    :ref:`convert your private key to OpenSSH format <converting PuTTY keys>`
+    or :ref:`generate a new key pair with MobaXterm <generating_keys_mobaxterm>`.
+
+Go to the `MobaXterm website <https://mobaxterm.mobatek.net>`__ and download the
+free version. Make sure to select the **Portable edition** from the download
+page. Create a ``MobaXterm`` folder in your ``Documents`` folder and
+uncompress inside it the contents of the downloaded zip file.
+
+We chose to create the MobaXterm folder on that particular location and we will
+refer to that in this guide. You can, of course, select some other location of
+your preference if you wish.
+
+.. note::
+
+    We will use ``my-vsc-cluster.example.com`` as a generic dummy address
+    throughout the guide. When trying to follow the steps on your machine
+    replace the address by a real remote host of the :ref:`VSC cluster <hardware>`
+    you wish to use.
```

Comment:
I think that most users will get lost with this. Reading the note before the instructions is mildly confusing and there will be users trying to configure the connection with `my-vsc-cluster.example.com`. So I suggest to remove this note and adapt the instructions below.

Reply:
ok

Reply:
ok